.github/workflows/build.yml

```diff
@@ -19,13 +19,19 @@ jobs:
                 publish-phar: [false]
                 include:
                     -   php: "7.3"
+                        composer-version: "1"
                         coverage: "pcov"
                         publish-phar: true
                     -   php: "7.4"
                         composer-version: "2"
 
+        env:
+            COMPOSER_ROOT_VERSION: '0.13.99'
+
         steps:
             -   uses: actions/checkout@v2
+                with:
+                    fetch-depth: 0
```

Comment:
With fetching all the history the `box compile` works

Reply:
It can be because composer uses git for checking the installed versions or something which is used in Box

Reply:
It is needed only for the phar build so it can be configurable via the matrix section

Reply:
👍

Reply:
Done